tests/test_workers/test_celery_logging.py

```diff
@@ -1,28 +1,80 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 import logging
+import os
 from unittest import mock
 
 from celery.utils.log import ColorFormatter
 
 from cachito.workers import celery_logging
 
 
-def test_setup_task_logging(caplog):
-    # Setting the logging level via caplog.set_level is not sufficient. The Flask
-    # related settings from previous tests interfere with this.
-    workers_logger = logging.getLogger("cachito.workers")
-    workers_logger.disabled = False
-    workers_logger.setLevel(logging.INFO)
+def test_cleanup_task_logging(tmpdir):
+    # Add a file handler first to test remove handler function
+    request_log_handler = logging.FileHandler("fake-path")
+    logger = logging.getLogger()
+    logger.addHandler(request_log_handler)
+
+    file_handler_count = 0
+    for handler in logger.handlers:
+        if isinstance(handler, logging.FileHandler):
+            file_handler_count = 1
+            break
+    # Assert that file handler was created
+    assert file_handler_count == 1
```

Comment:
This is essential testing that the Python standard library is working as expected. This is unnecessary in my opinion and can be removed.

Reply:
Hmmm in that case, I don't know what else to do to test `cleanup_task_logging` function. Also, if we are targeting 90% code coverage then might as well keep it?

Reply:
You definitely want to keep the test, I just meant you could remove the code block in this comment:
https://github.com/release-engineering/cachito/pull/215/files#r465956023